blogposts/2020/release-3.24.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Sourcegraph 3.24 release"
+publishDate: 2021-01-20T10:00-07:00
+description: "Sourcegraph 3.24 is released."
+tags: [blog, release]
+slug: "release/3.24"
+published: true
+changelogItems:
+  - description: Added panels to the Sourcegraph monitoring dashboards
+    url: https://docs.sourcegraph.com/admin/observability/metrics#grafana
+    category: Admin
+  - description: Suggested filters in the search results page can now be scrolled.
+    url: https://github.com/sourcegraph/sourcegraph/pull/17097
+    category: Search
+  - description: Dashboard links now included in monitoring alerts.
+    url: https://docs.sourcegraph.com/admin/observability/alerting
+    category: Admin
+  - description: "Added `serviceKind` field of the `ExternalServiceKind` type to `Repository.externalURLs` GraphQL API, `serviceType` field is deprecated and will be removed in the future releases."
```

Comment:
The `serviceType` field is now deprecated and will be removed completely in a future release. The `serviceKind` field of the `ExternalServiceKind` type has been added to `Repository.externalURLs` GraphQL API. 

@abschu this one is clunky trying to get it into active voice. @Joelkw can you confirm I didn't accidently muddy the meaning on this sentence by turning it into two sentences?

Reply:
@content-hacker I think meaning is preserved, I wrote that changelog line originally 😄 
I'll make sure to use active voice from now on 🙇

Reply:
Thanks for confirming! I've made the update.